common/lib/xmodule/xmodule/data.py

```diff
@@ -0,0 +1,25 @@
+"""
+Public data structures for this app.
+
+See OEP-49 for details
```

Comment:
It feels a little weird to me to have a `data.py` that's shared by all of XModule, and in particular to have certs stuff live there. It kinda feels like putting the cart ahead of the horse–it feels like a piece of specification that belongs to the cross-cutting concept of certificates, and should live somewhere over there. But I don't have a really good answer for where exactly to put it... going to mull on this for a bit...

Reply:
Yeah, let's leave this for now. It's not a bad spot, it doesn't introduce any dependency problems, and we can shift it later if there's a better home for it.